app/src/modules/files/routes/add-new.vue

```diff
@@ -13,19 +15,45 @@ const router = useRouter();
 
 const isOpen = useDialogRoute();
 
+let uploadController: any = null;
```

Comment:
This dependency binding between parent modal and nested upload component feels a little odd in Vue land. As we discussed, lets drop the manual pause/upload for now and revisit this at a later date